checkov/common/bridgecrew/platform_integration.py

```diff
@@ -73,6 +67,24 @@ def __init__(self):
         self.ckv_to_bc_id_mapping = None
         self.use_s3_integration = False
         self.platform_integration_configured = False
+        self.setup_http_manager(os.getenv('BC_CA_BUNDLE', None))
```

Comment:
I believe this call is unnecessary, because we will always call `setup_http_manager` directly from `main` with the CLI arg (which could be from the environment variable).

Reply:
I updated the PR to both check and skip if the `self.http` instance variable (a ProxyManager/PoolManager  object) is already defined, and to minimize the number of calls to `self.setup_http_manager` down to (hopefully) one ... while allowing the equivalent `BC_CA_BUNDLE` environment variable to establish a default if no `--ca/--ca-certificate` is specified. 

The complication here was: the former global `http` variable was defined when `bc_integration` was imported ... which occurs before `parser.parse_args` in `main.py` ... and also oocurs other integrations using based on the `BaseIntegrationFeature` class.